cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -43,21 +43,80 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
     // `appToggles` tells us whether a feature is toggled on or off in the current environment.
     if (appToggles.mediaPlayer.enabled) {
       describe('Media Player', () => {
-        it('should render a placeholder image', () => {
-          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-            ({ body }) => {
-              // `video` blocks can also contain audio.
-              const media = getBlockData('video', body);
-              if (media) {
-                cy.get('div[class^="StyledVideoContainer"]').within(() => {
-                  cy.get('amp-img')
-                    .should('have.attr', 'src')
-                    .should('not.be.empty');
-                });
-              }
-            },
-          );
-        });
+        // This test is being temporarily throttled to the service 'news'.
+        if (service === 'news') {
+          it('should have a Media Player', () => {
+            cy.viewport(600, 1008);
+            cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+              ({ body }) => {
+                const media = getBlockData('video', body);
+                if (media) {
+                  cy.get('div[class^="StyledVideoContainer"]').then(() => {
+                    cy.get('iframe[class^="i-amphtml-fill-content"]').then(
+                      $iframe => {
+                        cy.wrap($iframe.prop('contentWindow'), {
+                          // `timeout` only applies to the methods chained below.
+                          // `its()` benefits from this, and will wait up to 8s
+                          // for the mediaPlayer instance to become available.
+                          timeout: 8000,
+                        })
+                          .its('embeddedMedia.playerInstances.mediaPlayer')
+                          .invoke('currentTime')
```

Comment:
Do we need to check if `currentTime` is greater than `0` to determine whether a page has "...a Media Player"? If not, it might be more appropriate to use the `ready` property instead or, if we're just checking for the existence of a player, we could use the DOM.

Reply:
Not used the `ready` property before but i i do not ind looking into it and having a go at it